tests/s2n_test.h

```diff
@@ -76,12 +76,12 @@ int test_count;
         S2N_TEST_OPTIONALLY_ENABLE_FIPS_MODE();                     \
     } while(0)
 
-#define END_TEST_NO_INIT()                                          \
-    do {                                                            \
-        EXPECT_SUCCESS_WITHOUT_COUNT(s2n_in_unit_test_set(false));  \
-        END_TEST_PRINT()                                            \
-        return 0;                                                   \
-    } while(0)
+#define END_TEST_NO_INIT()                                         \
+    do {                                                           \
+        EXPECT_SUCCESS_WITHOUT_COUNT(s2n_in_unit_test_set(false)); \
+        END_TEST_PRINT()                                           \
+        return 0;                                                  \
+    } while (0)
```

Comment:
nit: Not sure if this change was intended. If you want to change the formatting here the other test init `#defines` should probably also be changed.